src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -116,20 +120,28 @@ export const useSwapper = () => {
     })
 
     if (!minimumQuote) return
+    //use gas estimate from quote if estimatedGasFees var is not populated yet
+    const quoteEstimatedGasFees = fromBaseUnit(minimumQuote?.feeData?.fee || 0, feeAsset.precision)
+
+    // to account for gas fees when trading Max ETH
+    const gasDeduction =
+      sellAsset && feeAsset && feeAsset.caip19 === sellAsset.currency.caip19
+        ? bnOrZero(estimatedGasFees || quoteEstimatedGasFees)
```

Comment:
We still want to accept a result of `0` from `estimatedGasFees` as valid, yeah?

```suggestion
        ? bnOrZero(estimatedGasFees ?? quoteEstimatedGasFees)
```